packages/eslint-plugin/tests/rules/prefer-standalone-component/cases.ts

```diff
@@ -0,0 +1,165 @@
+    })
+    class Test {}
+  `,
+];
+
+export const invalid = [
+  convertAnnotatedSourceToFailureCase({
+    description:
+      'should fail when a component does not have the standalone property in the decorator',
+    annotatedSource: `
+        @Component({})
+        ~~~~~~~~~~~~~~
+        class Test {}
+      `,
+    messageId,
+    suggestions: [
+      {
+        messageId: suggestAddStandalone,
+        output: `
+        @Component({standalone: true})
+        
+        class Test {}
+      `,
+      },
+    ],
+  }),
+  convertAnnotatedSourceToFailureCase({
+    description:
+      'should fail when a component has the standalone property set to false in the decorator',
+    annotatedSource: `
+        @Component({ standalone: false })
+        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        class Test {}
+      `,
+    messageId,
+    suggestions: [
+      {
+        messageId: suggestAddStandalone,
+        output: `
+        @Component({ standalone: true })
+        
+        class Test {}
+      `,
+      },
+    ],
+  }),
+  convertAnnotatedSourceToFailureCase({
+    description:
+      'should fail when a component has the standalone property set to false in a decorator with multiple properties',
+    annotatedSource: `
+        @Component({
+        ~~~~~~~~~~~~
+        standalone: false,
+        ~~~~~~~~~~~~~~~~~~
+        template: '<div></div>'
+        ~~~~~~~~~~~~~~~~~~~~~~~
+        })
+        ~~
+        class Test {}
+`,
+    messageId,
+    suggestions: [
+      {
+        messageId: suggestAddStandalone,
+        output: `
+        @Component({
+        
+        standalone: true,
+        
+        template: '<div></div>'
+        
+        })
+        
+        class Test {}
+`,
+      },
+    ],
+  }),
+  convertAnnotatedSourceToFailureCase({
+    description:
+      'should fail when a component has no standalone property in a decorator with one property',
+    annotatedSource: `
+        @Component({
+        ~~~~~~~~~~~~
+        template: '<div></div>'
+        ~~~~~~~~~~~~~~~~~~~~~~~
+        })
+        ~~
+        class Test {}
+`,
+    messageId,
+    suggestions: [
+      {
+        messageId: suggestAddStandalone,
+        output: `
+        @Component({
+        
+        standalone: true,template: '<div></div>'
+        
+        })
+        
+        class Test {}
+`,
+      },
+    ],
+  }),
+  convertAnnotatedSourceToFailureCase({
+    description:
+      'should fail when a component has no standalone property in a decorator with multiple properties',
+    annotatedSource: `
+        @Component({
+        ~~~~~~~~~~~~
+        selector: 'my-selector',
+        ~~~~~~~~~~~~~~~~~~~~~~~
+        template: '<div></div>'
+        ~~~~~~~~~~~~~~~~~~~~~~~
+        })
+        ~~
+        class Test {}
+`,
+    messageId,
+    suggestions: [
+      {
+        messageId: suggestAddStandalone,
+        output: `
+        @Component({
+        
+        standalone: true,selector: 'my-selector',
```

Comment:
Is the rule supposed to also ensure the code is indented ? I assumed it was `prettier` or another code formatter job

Reply:
I agree!